src/components/Disputes/DisputeTimeline.js

```diff
@@ -164,33 +182,116 @@ function ItemStep({ item, index, roundStepContainer }) {
   )
 }
 
+function Outcome({ outcome, phase }) {
+  const theme = useTheme()
+  const title =
+    phase && phase === DisputePhase.RevealVote ? 'JURY OUTCOME' : 'OUTCOME'
```

Comment:
We could use `text-transform: uppercase` instead of uppercase text:

```suggestion
    phase && phase === DisputePhase.RevealVote ? 'Jury outcome' : 'Outcome'
```